packages/ferric/package.json

```diff
@@ -13,15 +13,19 @@
     "ferric": "./bin/ferric.js"
   },
   "scripts": {
-    "start": "tsx src/run.ts"
+    "start": "tsx src/run.ts",
+		"test": "vitest"
   },
   "dependencies": {
-    "@napi-rs/cli": "~3.0.3",
     "@commander-js/extra-typings": "^13.1.0",
+    "@napi-rs/cli": "~3.0.3",
     "bufout": "^0.3.2",
     "chalk": "^5.4.1",
     "commander": "^13.1.0",
-    "react-native-node-api": "0.3.2",
-    "ora": "^8.2.0"
+    "ora": "^8.2.0",
+    "react-native-node-api": "0.3.2"
+  },
+  "devDependencies": {
+    "vitest": "^3.2.4"
```

Comment:
Could we take out the tests (and addition of `vitest`) from this PR to simplify the review? Feels more like they belong in an independent PR.

Reply:
Please be mindful of the technology choices already made in the project and help keep things converged. 

We're using the Node.js built-in test runner via tsx (`tsx --test`) in the other packages. (And I'm not looking for a debate on which is the superior choice in a PR review comment 🙈)